zilencer/migrations/0038_remove_remoteinstallationcount_zilencer_remoteinstallat_server_id_remote_id_f72e4c30_idx_and_more.py

```diff
@@ -0,0 +1,34 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zilencer", "0037_alter_remoteinstallationcount_unique_together_and_more"),
+    ]
```

Comment:
It's probably worth giving this migration a human-chosen name so the filename isn't so ridiculously long.